### ISSUE TITLE
MRPC Produces Two Vastly Different Eval Accuracy

### ISSUE THREAD

**Issue by wei-v-wang** [google-research/bert#1037]
Hi, 

I am testing using TF v1.14.0. 
I ran for 10 times with run_classifier.py (do_traing=False, do_eval=True), i.e. do inference immediately after loading https://storage.googleapis.com/bert_models/2018_10_18/uncased_L-24_H-1024_A-16.zip 

I am seeing two values 0.31617647 & 0.6838235. 
I wonder why I am seeing different inference results and why I am seeing exactly these two results. Thanks! 

check-pretrained-MRPC-1.log:INFO:tensorflow:  eval_accuracy = 0.31617647
check-pretrained-MRPC-10.log:INFO:tensorflow:  eval_accuracy = 0.6838235
check-pretrained-MRPC-2.log:INFO:tensorflow:  eval_accuracy = 0.31617647
check-pretrained-MRPC-3.log:INFO:tensorflow:  eval_accuracy = 0.6838235
check-pretrained-MRPC-4.log:INFO:tensorflow:  eval_accuracy = 0.31617647
check-pretrained-MRPC-5.log:INFO:tensorflow:  eval_accuracy = 0.31617647
check-pretrained-MRPC-6.log:INFO:tensorflow:  eval_accuracy = 0.31617647
check-pretrained-MRPC-7.log:INFO:tensorflow:  eval_accuracy = 0.31617647
check-pretrained-MRPC-8.log:INFO:tensorflow:  eval_accuracy = 0.31617647
check-pretrained-MRPC-9.log:INFO:tensorflow:  eval_accuracy = 0.31617647

**Comment by wei-v-wang** [comment]
Update: ran another 10 times and was able to see a 3rd accuracy value

check-pretrained-MRPC-1.log:INFO:tensorflow:  eval_accuracy = 0.31617647
check-pretrained-MRPC-10.log:INFO:tensorflow:  eval_accuracy = 0.6838235
check-pretrained-MRPC-2.log:INFO:tensorflow:  eval_accuracy = 0.6838235
check-pretrained-MRPC-3.log:INFO:tensorflow:  eval_accuracy = 0.6838235
check-pretrained-MRPC-4.log:INFO:tensorflow:  eval_accuracy = 0.31617647
check-pretrained-MRPC-5.log:INFO:tensorflow:  eval_accuracy = 0.6764706
check-pretrained-MRPC-6.log:INFO:tensorflow:  eval_accuracy = 0.6838235
check-pretrained-MRPC-7.log:INFO:tensorflow:  eval_accuracy = 0.31617647
check-pretrained-MRPC-8.log:INFO:tensorflow:  eval_accuracy = 0.6838235
check-pretrained-MRPC-9.log:INFO:tensorflow:  eval_accuracy = 0.31617647

and also read that " Small sets like MRPC have a high variance in the Dev set accuracy, even when starting from the same pre-training checkpoint." from README. Hence closing. :)

**Comment by ClarkChin08** [comment]
i also read this word "Small sets like MRPC have a high variance in the Dev set accuracy, even when starting from the same pre-training checkpoint", but how could this happened, i got the same input features and same weight, after inference got different probabilities? **same input but different ouput!**
